pt_os_web_portal/pt-os-web-portal

```diff
@@ -7,7 +7,7 @@ import pitop.common.ptdm  # noqa: E402
 import zmq.green  # noqa: E402
 
 # Use zmq.green for gevent compatibility
-pitop.common.ptdm.zmq = zmq.green
+pitop.common.ptdm.import_zmq = lambda: zmq.green
```

Comment:
could be worth keeping both forms of this in case we change it again and directly import zmq